code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -318,12 +318,12 @@
 			module_sprites["Basic"] = "secborg"
 			module_sprites["Blue Knight 2.0"] = "sleeksecurity"
 			module_sprites["Black Knight"] = "securityrobot"
-			module_sprites["Bloodhound"] = "bloodhound"
+			module_sprites["Hound"] = "bloodhound"
```

Comment:
It still sounds like it's better suited to hunting criminals than it is at keeping the peace.